adserver/models.py

```diff
@@ -175,6 +175,12 @@ class Publisher(TimeStampedModel, IndestructibleModel):
     record_placements = models.BooleanField(
         default=False, help_text=_("Record placement impressions for this publisher")
     )
+    render_pixel = models.BooleanField(
+        default=True,
```

Comment:
Actually, I wonder if this should default False, so all new publishers need to use the client, and we can enable it if they want a custom integration. 

I think for now we default True for the migration, but eventually switch to default False